opm/parser/eclipse/EclipseState/Schedule/Well.hpp

```diff
@@ -86,6 +86,10 @@ namespace Opm {
         WellPolymerProperties          getPolymerPropertiesCopy(size_t timeStep) const;
         const WellPolymerProperties&   getPolymerProperties(size_t timeStep) const;
 
+        bool                           setSolventFraction(size_t timeStep , const double fraction);
+        double                         getSolventFractionCopy(size_t timeStep) const;
```

Comment:
No - as long as the object stored in the `DynamicState<T>` is only a scalar double that is not needed.